client/flutter/lib/components/page_scaffold/page_scaffold.dart

```diff
@@ -0,0 +1,52 @@
+import './page_header.dart';
+import './share_bar.dart';
+import 'package:flutter/material.dart';
+
+class PageScaffold extends StatelessWidget {
+  final String title;
+  final String subtitle;
+
+  final List<Widget> body;
+
+  final BuildContext context;
```

Comment:
@theswerd You removed the `context` from `PageHeader` (👍), but not from this file, `PageScaffold` 😛 ...